corehq/apps/hqwebapp/static/cloudcare/scss/formplayer-webapp/form.scss

```diff
@@ -131,6 +131,15 @@ $group-indent: 15px;
       height: 1em;
       vertical-align: baseline;
     }
+
+    .btn {
+      background-color: white;
```

Comment:
Does it just not look good with the background being red?

Reply:
Before this PR, it appears as white because the button's background is transparent. With the underlying background changing to blue, the button also becomes blue if i don't make it white.